package.json

```diff
@@ -26,13 +26,13 @@
         "start": "ng serve dev",
         "clean": "del ./dist",
         "test:aot": "ng build --prod ks-app",
-        "test:format": "./scripts/clang-check.sh",
+        "test:format": "node ./scripts/format-check.js",
         "test:lint": "tslint -p . -c tslint.json \"src/**/*.ts\" -e \"src/ks-app/**\" -e \"src/schematics/**\"",
         "test:visual": "./scripts/docker-cdc.js -t",
         "test": "ng test clr-angular",
         "test:watch": "ng test clr-angular --watch",
-        "test:travis": "npm-run-all build:ui test:format test:lint test test:aot",
-        "format:fix": "node_modules/.bin/clang-format -style=file -i --glob=src/**/*.ts",
+        "test:travis": "npm-run-all build:ui test:lint test test:aot",
```

Comment:
is there an equivalent of `test:format` in the prettier version? that is, make the travis build fail if the code isn't formatted to prettier's standard?

Reply:
Oh I might have spoken too soon. Is that handled by husky?

Reply:
I have to disable it until all of the files are formatted. So its not enabled for this commit. Changed files will be formatted with a `pre-commit` hook. 

And, once this goes in the bulk formatting  per file type will follow within an hour or two. I'll re-enable the `pre-push` hook that checks for correct formatting and add the script back to the test:travis job.

Reply:
@jeeyun - fyi husky only enables git hooks so that we can define them in the package.json as `npm scripts` instead of mucking with the `./.git/hooks` files themselves like we did w/ clang checking.